wifiphisher/common/phishinghttp.py

```diff
@@ -20,6 +20,24 @@ def get(self):
         self.redirect("http://10.0.0.1:8080/")
 
 
+class ValidateHandler(tornado.web.RequestHandler):
+
+    def initialize(self, em):
+        self.em = em
+
+    def get(self, cred_type, cred_content):
+        """
+        Override the get method
+
+        :param self: A tornado.web.RequestHandler object
+        :type self: tornado.web.RequestHandler
+        :return: None
+        :rtype: None
+        """
+        value = self.em.verify_cred((cred_type, cred_content))
+        self.write("%s" % value)
+
```

Comment:
@sophron I'm not so sure how does the extension render a new page for the users based on the result of verification?

For example, if the passphrase was wrong given by the users, I could return a string `"The passphrase is wrong!"` from `verify_cred` and `self.write` in the `ValidateHandler` will help render this string on the new page to the users?